### DISCUSSION TITLE
Scrollmagic and Flynt

### DISCUSSION THREAD

**Discussion by marcotiberio** [flyntwp/flynt#420] [Q&A] [answered]
Hello,

I have installed and imported Scrollmagic and I'd like to use it globally across my website without scoping it to selected components.
What would it be the best way to do that?

Best,
Marco

**Comment by steffenbew** [comment, marked as answer]
@marcotiberio Hi Marco, I think I would load it as a global script into the [assets/main.js](https://github.com/flyntwp/flynt/blob/master/assets/main.js) file. That's what we've done for handling the icons and lazy loading in Flynt.

**Reply by marcotiberio** [reply]
Thank you very much @steffenbew :)
